src/components/search/filters/AggregationFilterBucket/utils.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactElement, cloneElement } from 'react'
+import Highlighter from 'react-highlight-words'
+
+import { Bucket, SourceField } from '../../../../Types'
+import { getTagIcon, getTypeIcon } from '../../../../utils/utils'
+
+export const getTagsDisplayLabel = (icon: ReactElement, bucket: Bucket, quickFilter?: string) => (
```

Comment:
maybe create separate function components: `TagsDisplayLabel`, `FileTypeDisplayLabel`, `DisplayLabel` and put them in separate files?

Reply:
good point, moved them in separate components